src/modules/hide_sidebar_elements/index.js

```diff
@@ -28,6 +52,20 @@ class HideSidebarElementsModule {
         $('body').toggleClass('bttv-hide-featured-channels', settings.get('hideFeaturedChannels'));
     }
 
+    toggleAutoExpandChannels() {
+        if (settings.get('autoExpandChannels') === true) {
+            $('.side-nav-load-more__button').trigger('click');
+        }
+    }
+
+    toggleRecommendedFriends() {
+        $('body').toggleClass('bttv-hide-recommended-friends', settings.get('hideRecommendedFriends'));
+    }
+
+    toggleOfflineFollowedChannels() {
+        $('body').toggleClass('bttv-hide-followed-offlines', settings.get('hideOfflineFollowedChannels'));
```

Comment:
I believe this is supposed to be `bttv-hide-followed-offline` and not `bttv-hide-followed-offlines` ?

Reply:
yes, I'm bad at words